src/controllers/GeneratorTrait.php

```diff
@@ -58,6 +59,9 @@ private function generateResources(): void
             $this->tests = new Tests($this);
             $this->tests->createEntity($this->formatFuncTestsPath(), DefaultInterface::FUNCTIONAL_POSTFIX);
         }
+
+        FileManager::createPath($this->formatFuncTestsPath());
```

Comment:
It would probably better to place this line right after if statement 
```php
         if (empty($this->options[ConsoleInterface::OPTION_TESTS]) === false) {
```

Reply:
Ok!

Reply:
Great, thanks a lot for contribution.